canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -80,21 +85,49 @@ class CheckboxsetControl extends React.Component {
 		}
 		const checkboxes = [];
 		for (var i = 0; i < this.props.control.values.length; i++) {
+			const tooltipId = uuid4() + "-tooltip-" + this.props.control.name;
+			let tooltip = "";
+			if (this.props.control.description && !(this.props.state === STATES.DISABLED || this.props.state === STATES.HIDDEN) && !this.props.tableControl) {
```

Comment:
Why have the disabled/hidden checks here?  Wouldn't the disable check go on the tooltip?

Reply:
I did add it within the Tooltip complement.